spec/models/search/work_query_spec.rb

```diff
@@ -177,46 +177,115 @@
   end
 
   describe "#works_per_language" do
-    it "returns the count of works per language code" do
-      es_response = {
-        "took" => 16,
-        "timed_out" => false,
-        "_shards" => {
-          "total" => 12,
-          "successful" => 12,
-          "skipped" => 0,
-          "failed" => 0
-        },
-        "hits" => {
-          "total" => { "value" => 10_000, "relation" => "gte" },
-          "max_score" => nil,
-          "hits" => []
-        },
-        "aggregations" => {
-          "languages" => {
-            "doc_count_error_upper_bound" => 0,
-            "sum_other_doc_count" => 0,
-            "buckets" => [
-              { "key" => "en", "doc_count" => 863_269 },
-              { "key" => "ru", "doc_count" => 11_476 },
-              { "key" => "zh", "doc_count" => 1622 }
-            ]
+    context "when invoked by a guest" do
+      it "returns the count of works per language code" do
```

Comment:
Can we be more specific with our descriptions to make it easier to see at a glance what the difference is between guest and logged in behavior? e.g., `returns the count of public, revealed, and unhidden works per language code"